spec/tasks/gen/model_spec.cr

```diff
@@ -9,21 +9,33 @@ describe Gen::Model do
       Gen::Migration.silence_output do
         io = IO::Memory.new
         model_name = "ContactInfo"
-        ARGV.push(model_name)
+        ARGV.push(model_name, "name:String", "contacted_at:Time")
```

Comment:
I think there should also be a test that just passes the model name and no columns, just to make sure that works too

Reply:
Will do!

Reply:
Good you noticed because that didn't work. It's fixed now.